drivers/homeless_summary_report/spec/models/report_context.rb

```diff
@@ -0,0 +1,55 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+RSpec.shared_context 'report context', shared_context: :metadata do
+  def default_setup_path
+    'drivers/homeless_summary_report/spec/fixtures/files/default'
+  end
+
+  def default_filter
+    {
+      start: Date.parse('2022-01-01'),
+      end: Date.parse('2022-12-31'),
+      project_type_codes: GrdaWarehouse::Hud::Project::RESIDENTIAL_PROJECT_TYPES.keys,
+      coc_codes: ['XX-500', 'XX-501'],
+    }
+  end
+
+  def coc_filter
+    {
+      start: Date.parse('2022-01-01'),
+      end: Date.parse('2022-12-31'),
+      project_type_codes: GrdaWarehouse::Hud::Project::RESIDENTIAL_PROJECT_TYPES.keys,
+      coc_codes: ['XX-501'],
+    }
+  end
+
+  def run!(filter)
+    HomelessSummaryReport::Report.new(user_id: User.setup_system_user.id, filter: filter).run_and_save!
+  end
+
+  def report_result
+    HomelessSummaryReport::Report.last
+  end
+
+  def result(field, demographics)
+    HomelessSummaryReport::Result.find_by(field: field, detail_link_slug: demographics, calculation: :count, report_id: report_result.id).value
+  end
```

Comment:
Maybe this should be a method on `HomelessSummaryReport::Report` (give me the result for this field and demographics?
```
def result_count(field, demographics)
  results.find_by(field: field, detail_link_slug: demographics, calculation: :count)
end
```

Reply:
I didn't do this, without knowing if I was just changing the report for the benefit of the tests.

Reply:
yeah, it would just be for the benefit of the tests, no need to do it, just seemed like it might be useful.